tests/features.py

```diff
@@ -0,0 +1,41 @@
+from flask import current_app
+from mock import patch
+
+
+def enable_feature(name, value=True):
```

Comment:
the above code will throw an error about flasks application context being accessed out of scope...